hazelcast-sql/src/test/java/com/hazelcast/jet/sql_nightly/SqlSTSInnerEquiJoinFaultToleranceStressTest.java

```diff
@@ -151,12 +171,16 @@ public void setUp() throws Exception {
         sqlService.execute("CREATE VIEW s2 AS " +
                 "SELECT * FROM TABLE(IMPOSE_ORDER(TABLE " + sourceTopic + " , DESCRIPTOR(__key), 4))");
 
-        jobRestarter = new JobRestarter(instance());
+        jobRestarter = new JobRestarter(coordinator);
         jobRestarter.start();
     }
 
     @After
     public void after() throws InterruptedException {
+        // https://hazelcast.atlassian.net/browse/HZ-3187
+        if (Objects.equals(processingGuarantee, EXACTLY_ONCE) && !restartGraceful) {
```

Comment:
why not remove the values from the test parameters?

Reply:
Because we want to test both `EXACTLY_ONCE` with graceful shutdown and `AT_LEAST_ONCE` with forceful terminaiton. Also, we plan to fix the mentioned issue and remove this check.

Reply:
it is easier to use `assumeTrue` - you wouldn't have to repeat the check. But it is acceptable here as this is a backport.